lib/view/components/messaging/chat_bubble_component.dart

```diff
@@ -32,15 +32,16 @@ class ChatBubble extends StatelessWidget {
             message,
             style: TextStyle(
               fontSize: 20,
-              color: Theme.of(context).colorScheme.surface,
-              fontWeight: FontWeight.bold
+              color: Theme.of(context).colorScheme.onSurface,
```

Comment:
Just a question, what is the difference between onSurface and surface?

Reply:
So surface uses the surface color. Which in this case is white, but that would make the text illegible. So onSurface makes sure to use a color that is legible on the surface it is on. So it uses black for the text in light mode and in dark mode too.